src/dbt/kipptaf/models/surveys/intermediate/int_surveys__survey_responses.sql

```diff
@@ -54,6 +54,7 @@ left join
     on rt.name = fr.info_title
     and date(fr.last_submitted_time) between rt.start_date and rt.end_date
     and eh.assignment_status not in ('Terminated', 'Deceased')
+where fr.rn_form_item_respondent_submitted_desc = 1
```

Comment:
Add this as a column and filter it downstream. There might be instances where we want multiple responses to a form.